main-module/src/main/java/com/diffblue/javatest/StringStuff.java

```diff
@@ -0,0 +1,346 @@
+package com.diffblue.javatest;
+
+import java.io.PrintStream;
+import java.util.Locale;
+
+public class StringStuff {
+
+  public static String classGetNameExample(Class c) {
```

Comment:
@peterjwest The problem is that the names then become very close or identical to the things they are using.